cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/core/ticket/ProxyGrantingTicketProperties.java

```diff
@@ -0,0 +1,20 @@
+package org.apereo.cas.configuration.model.core.ticket;
+
+/**
+ * This is {@link ProxyGrantingTicketProperties}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+
+public class ProxyGrantingTicketProperties {
+    private int maxLength = 50;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['50' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2769871173/issues/source?bid=3256115&fileBranchId=3357337#l11)